code/__defines/species_languages.dm

```diff
@@ -54,6 +54,7 @@
 #define LANGUAGE_TERMINATOR "Hephaestus Darkcomms"	// HKs.
 
 // Lesser-form Languages
+#define BRAZILIAN "Gibbering"				// alberyk
```

Comment:
`#define LANGUAGE_GIBBERING` you mean.

Reply:
This was changed to `#define LANGUAGE_BRAZILIAN`. That's unacceptable. It needs to be `#define LANGUAGE_GIBBERING`, unless you're gunning for a "most unmaintainable code" award.